src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scenes/SpectatorView.ExtrinsicsCalibration.unity

```diff
@@ -473,11 +426,9 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 7abda130a4f4dfc4989cb445d1bd8fae, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  cameraIntrinsicsPath: C:\Users\chriba\Documents\Calibration\CameraIntrinsics_1.json
-  expectedNumberOfMarkers: 18
-  MatchMakingService: {fileID: 740770235}
-  NetworkingService: {fileID: 740770235}
-  headsetCalibration: {fileID: 0}
+  cameraIntrinsicsPath: C:\Users\chriba\documents\Calibration\CameraIntrinsics_3.json
```

Comment:
>C:\Users\chriba\documents\Calibration\CameraIntrinsics_3.json [](start = 24, length = 61)

Probably don't want a path on your dev machine to be checked into the public repo